src/utils.js

```diff
@@ -11,13 +11,13 @@ export const listenerOptions = {
 
 export function preventTouchMove(e) {
   e.preventDefault();
-  
+
   return false;
 }
 
 export function allowTouchMove(e) {
   const target = e.currentTarget;
-  
```

Comment:
My linter removed the extra spaces in this line. Hope it's alright. Same as above.